### PR TITLE
Replace bullet points

### PR DESCRIPTION
**Pull request by abeatrix**
Replace bullet points with a numbered list for steps
Update languages used
Add a link on Configure cluster access

### CODE REVIEW

File: doc/admin/install/kubernetes/index.md [hunk 1]
```diff
@@ -14,79 +14,80 @@ The Kubernetes manifests for a Sourcegraph on Kubernetes installation are in the
   - Sourcegraph requires an SSD backed storage class
   - [Cluster role administrator access](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) v1.15 or later
+  - [Configure cluster access](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/) for `kubectle`
```
Comment (daxmc99) [inline doc/admin/install/kubernetes/index.md:17]:
```suggestion
  - [Configure cluster access](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/) for `kubectl`
```

File: doc/admin/install/kubernetes/index.md [hunk 2]
```diff
@@ -14,79 +14,80 @@ The Kubernetes manifests for a Sourcegraph on Kubernetes installation are in the
...
 
 > WARNING: You need to create a [fork of our deployment reference.](configure.md#fork-this-repository)
 
 ## Steps
 
-- Make sure you have configured `kubectl` to [access your cluster](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+1. After meeting all the requirements, make sure you can [access your cluster](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) with `kubectle`.
```
Comment (daxmc99) [inline doc/admin/install/kubernetes/index.md:23]:
```suggestion
1. After meeting all the requirements, make sure you can [access your cluster](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) with `kubectl`.
```

File: doc/admin/install/kubernetes/index.md [hunk 3]
```diff
@@ -14,79 +14,80 @@ The Kubernetes manifests for a Sourcegraph on Kubernetes installation are in the
...
 
-   kubectl 1.10.0 or later:
+   - kubectl 1.9.x:
 
-   ```
-   kubectl port-forward svc/sourcegraph-frontend 3080:30080
-   ```
+      ```bash
+      kubectl port-forward $(kubectl get pod -l app=sourcegraph-frontend -o template --template="{{(index .items 0).metadata.name}}") 3080
+      ```
```
Comment (daxmc99) [inline doc/admin/install/kubernetes/index.md:73]:
```suggestion
```
This is quite old and we don't support this k8s version

File: doc/admin/install/kubernetes/index.md [hunk 4]
```diff
@@ -14,79 +14,80 @@ The Kubernetes manifests for a Sourcegraph on Kubernetes installation are in the
...
 
-   ```bash
-   kubectl port-forward $(kubectl get pod -l app=sourcegraph-frontend -o template --template="{{(index .items 0).metadata.name}}") 3080
-   ```
+      ```
+      kubectl port-forward svc/sourcegraph-frontend 3080:30080
+      ```
 
-   kubectl 1.10.0 or later:
+   - kubectl 1.9.x:
```
Comment (daxmc99) [inline doc/admin/install/kubernetes/index.md:69]:
```suggestion
```

**Review by davejrt — APPROVED** [review]

**Pull request merged**